src/main/java/lotto/domain/Price.java

```diff
@@ -0,0 +1,46 @@
+package lotto.domain;
+
+import java.util.List;
+import java.util.Objects;
+
+public class Price {
+    private static final int LOTTO_TICKET_PRICE = 1000;
+    private int price;
+
+    public Price(int price) {
+        validatePrice(price);
+        this.price = price;
+    }
+
+    private void validatePrice(int price) {
+        if (price < 1000) {
```

Comment:
아래의 코드로 개선해 보면 어떨까요?

```suggestion
        if (price < LOTTO_TICKET_PRICE) {
```